WalletWasabi/Helpers/NBitcoinHelpers.cs

```diff
@@ -75,6 +75,24 @@ public static ExtPubKey BetterParseExtPubKey(string extPubKeyString)
 			return epk;
 		}
 
+		public static HDFingerprint BetterParseHDFingerprint(string hdFingerPrintString)
+		{
+			hdFingerPrintString = Guard.NotNullOrEmptyOrWhitespace(nameof(hdFingerPrintString), hdFingerPrintString, trim: true);
+
+			HDFingerprint hdfp;
+			try
+			{
+				var hdfpu = uint.Parse(hdFingerPrintString);
+				hdfp = new HDFingerprint(hdfpu);
+			}
+			catch
+			{
+				// Try hex, Old wallet format was like this.
+				hdfp = new HDFingerprint(ByteHelpers.FromHex(hdFingerPrintString));
```

Comment:
unsure why you changed the format. Bitcoin Core and coldcard both use hex.

Reply:
We are showing transaction hex, not PSBT Hex. We show PSBT Base64 String. CC doesn't do PSBT String. So what you say is that Core and CC do PSBT Hex, right? That was never implemented. Anyway it'll be binary now (assuming it works, still didn't test.)